superset/db_engine_specs/crate.py

```diff
@@ -59,7 +59,7 @@ def convert_dttm(
         sqla_type = cls.get_sqla_column_type(target_type)
 
         if isinstance(sqla_type, types.TIMESTAMP):
-            return f"(({dttm.timestamp() * 1000})::BIGINT)"
+            return f"{dttm.timestamp()}"
```

Comment:
`f"""CAST('{dttm.isoformat()}' AS TIMESTAMP)"""`

is more in line with other solutions:
https://github.com/apache/superset/blob/2944d7b32ac6aaf1aaf33b9591911fdcbae25547/superset/db_engine_specs/postgres.py#L195

Reply:
@hlcianfagna do you agree? @proddata would you be able to help out reviewing and testing this PR? I don't use CrateDB right now, so any help validating this would be helpful.

Reply:
I think it is debatable, without knowing how Superset exactly uses this method. 

From the description of the [base class](https://github.com/apache/superset/blob/2944d7b32ac6aaf1aaf33b9591911fdcbae25547/superset/db_engine_specs/base.py#L1059-L1070) it seems like this method should convert a Python `datetime` object to the database target data type, which would be `TIMESTAMP`, however CrateDB can implicitly cast from a floating point value to `TIMESTAMP`, which is why the proposed solution of @hlcianfagna most likely also works.

Reply:
I tested @proddata 's suggestion and it works well, it also looks safer/more elegant so I made a new commit